status/private/networks.nim

```diff
@@ -25,7 +25,7 @@ type
     name*: string
 
 # set via `nim c` param `-d:INFURA_TOKEN:[token]`; should be set in CI/release builds
-const INFURA_TOKEN {.strdefine.} = ""
+const INFURA_TOKEN {.strdefine.} = "220a1abb4b6943a093c35d0ce4fb0732" # @TODO: remove this
```

Comment:
We can eventually supply the default via the Makefile, like we do in status-desktop.